_attributes/common-attributes.adoc

```diff
@@ -133,8 +133,8 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
```

Comment:
While the MaistraVersion and the SMProductVersion are both content variables, they are used in different places and serve slightly different purposes. The MaistraVersion should remain 2.3 or some code examples won't work.

I know this is confusing as you're learning :)